xml/System.Net.Http/HttpVersionPolicy.xml

```diff
@@ -34,7 +34,15 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Uses only the requested version. Throwing <see cref="HttpRequestException" /> if a connection with the exact version cannot be established.</summary>
```

Comment:
```suggestion
        <summary>Only use the requested version. Throws an <see cref="HttpRequestException" /> if a connection with the exact version cannot be established.</summary>
```

Question: Which method(s) would throw that exception? We should include that here, or a general description of those methods, because the enum member doesn't throw it.

Reply:
`SocketsHttpHandler` send methods since the property (`HttpRequestMessage.VersionPolicy`) is interpreted there. This also influences the behavior of `HttpClient` if it's used with default handlers. OTOH, it has no effect on `WinHttpHandler` for instance.

Reply:
The HttpClient has a HttpRequestException exception mentioned but in general way. On other side this description is very specific for SocketsHttpHandler and this policy.

I would leave it like it is or delete it from the `HttpVersionPolicy.xml` as the reason why it can be thrown is quite obvious.

@gewarren, what is your opinion there?

Reply:
I think just remove that part of the sentence. I'll edit my suggestion to reflect that.